django/db/models/expressions.py

```diff
@@ -519,7 +519,7 @@ def resolve_expression(self, query=None, allow_joins=True, reuse=None, summarize
             c.source_expressions[pos] = arg.resolve_expression(query, allow_joins, reuse, summarize, for_save)
         return c
 
-    def as_sql(self, compiler, connection, function=None, template=None):
+    def as_sql(self, compiler, connection, **kwargs):
```

Comment:
Is it best to remove common parameters like `function` and `template`? Seems to me it improves readability a bit to have common kwargs explicitly declared.